pkg/cache/fair_sharing.go

```diff
@@ -75,3 +80,30 @@ func dominantResourceShare(node dominantResourceShareNode, wlReq resources.Flavo
 	dws := drs * 1000 / node.fairWeight().MilliValue()
 	return int(dws), dRes
 }
+
+// calculateLendable aggregates capacity for resources across all
+// FlavorResources.
+func calculateLendable(node hierarchicalResourceNode) map[corev1.ResourceName]int64 {
+	// walk to root
+	root := node
+	for root.HasParent() {
+		root = root.parentHRN()
+	}
```

Comment:
nit: this could be simplified with `getRootUnsafe`, but feel free for follow up.

Reply:
ah no this is first level parent, forget